tests/shell/run_all.sh

```diff
@@ -10,7 +10,10 @@ cd "@CMAKE_INSTALL_PREFIX@/@TARGET_TOOL_EXEC_FOLDER@"
 
 nbFailed=""
 
-EXPORTS=`mktemp -d`
+alias mktempdir-elektra=`mktemp -d 2>/dev/null || mktemp -d -t 'libelektra-test'`
+alias mktempfile-elektra=`mktemp -t elektraenv.XXXXXXXXX  2>/dev/null || mktemp -t 'elektraenv'`
```

Comment:
did you test this? as far as I recall, some shells do not allow aliases in non-interactive mode.